onert-micro/onert-micro/include/pal/common/PALArithmeticOpCommon.h

```diff
@@ -141,6 +172,71 @@ OMStatus BroadcastArithmeticOp4DSlow(const core::BinaryArithmeticBroadcastParams
   return Ok;
 }
 
+template <typename T, typename Fn>
+OMStatus QuantizedBroadcastArithmeticOp4DSlow(
+  const core::BinaryArithmeticBroadcastParams &params, const core::OMRuntimeShape &input1_shape,
+  const onert_micro::core::QuantizationParams &input1_qparams, const T *input1_data,
+  const core::OMRuntimeShape &input2_shape,
```

Comment:
Not sure. But may it would be better to group `shape, q_params with input_data` into `QuantizedData`?